### DISCUSSION TITLE
[select] select下拉框在单选模式下使用 slot #tag 无效

### DISCUSSION THREAD

**Discussion by shachaba** [element-plus/element-plus#15692] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `select`

## Description

select组件在配置multiple 时, <template #tag>自定义内容</<template> 有效, 但去掉multiple多选时, 插槽#tag就没有效果

### Additional comments

希望select组件的这个tag插槽不管单选或多选都有效!

**Comment by chenxch** [comment]
多选的时候才会有tag
